### ISSUE TITLE
Regression (insiders): Python debugger fails in Fish shell

### ISSUE THREAD

**Issue by karrtikr** [Microsoft/vscode#68998]
@rpavlik commented on [Tue Feb 19 2019](https://github.com/Microsoft/vscode-python/issues/4444)

## Environment data
- VS Code version: 
> Version: 1.32.0-insider
> Commit: 003521e715f2dbcf4f98be99a923097505859954
> Date: 2019-02-15T06:13:33.916Z
> Electron: 3.1.3
> Chrome: 66.0.3359.181
> Node.js: 10.2.0
> V8: 6.6.346.32
> OS: Linux x64 4.19.0-0.bpo.2-amd64
- Extension version: 2019.1.0
- OS and version: Debian Stretch
- Python version: Using the Python 3.5.3 that comes with Stretch.
- Type of virtual environment used (N/A | venv | virtualenv | conda | ...): None or virtualenv
- Relevant/affected Python packages and their versions: n/a

fish version: 2.7.1

## Expected behaviour

I should (and used to be able to, possibly as recently as last week) be able to hit F5 and debug a Python script.

## Actual behaviour

The process of running the script sources the fish-appropriate virtualenv activate script, if not already done and if configured. Then, it tries running a command line that has a `&&`, which isn't fish syntax (ugh, bugs me as much as it bugs you). Sample command and output:

```
cd /home/ryan/src/vulkan && env PYTHONIOENCODING=UTF-8 PYTHONUNBUFFERED=1 /usr/bin/python3 /home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/pythonFiles/ptvsd_launcher.py --default --client --host localhost --port 36379 /home/ryan/src/vulkan/xml/genvk.py -registry xml/vk.xml -o validity validinc 
fish: Unsupported use of '&&'. In fish, please use 'COMMAND; and COMMAND'.
```

Eventually it ends up with a timeout, as shown in the screen capture below.  This did work in the very recent past.

![vscode-python-fish-screencast1](https://user-images.githubusercontent.com/61129/53034929-ef0e9000-3439-11e9-9e9a-7bd4075bc59c.gif)

I tried to work around this by launching bash in the terminal before starting the debug session, but it seems to start a new shell in that case.

![vscode-python-fish-screencast2](https://user-images.githubusercontent.com/61129/53034985-09486e00-343a-11e9-9e13-3db9ca8d4c3e.gif)

If I update my `terminal.integrated.shell.linux`to be `/bin/bash` instead of `/usr/bin/fish`, the same command works just fine, but I would really like to be able to use fish in VS Code in general for interactive use. When searching my settings for "shell" it looks like the Docker extension explicitly launches `sh` in this case - might be easier than supporting all the oddball shells out there. Not sure what changed recently to make this no longer work.

## Steps to reproduce:
1. Set fish as your shell - either for your user or just in vscode via `terminal.integrated.shell.linux`.
2. (optional) Set up a virtualenv, choose it as your python environment, and notice it is working fine - `. venv/bin/activate.fish` gets put into the terminal automatically.
3. Try running any python (3, didn't test 2) app through the debugger. The config snippet  I used for testing and screencast is below. (used with https://github.com/KhronosGroup/Vulkan-Docs - though it works with other projects too and is presumably independent of the code)

```json
        {
            "name": "Python: gen valid for issue",
            "type": "python",
            "request": "launch",
            "program": "${workspaceFolder}/xml/genvk.py",
            "console": "integratedTerminal",
            "args": [
                "-registry",
                "xml/vk.xml",
                "-o",
                "validity",
                "validinc",
            ]
        },
```

## Logs
Output for `Python` in the `Output` panel (`View`→`Output`, change the drop-down the upper-right of the `Output` panel to `Python`)

```
Starting Jedi Python language engine.
```

(The default selection here was remote-attach, but it shows nothing.)

Output from `Console` under the `Developer Tools` panel (toggle Developer Tools on under `Help`; turn on source maps to make any tracebacks be useful by running `Enable source map support for extension debugging`)

(I reloaded the window, with no documents open, then just hit F5 to start debugging, to get this trace... If it wasn't the first time I had tried to debug since a window-reload, and I just ran `clear()` before hitting F5, there was only one message shown at the default levels.)

```
[Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Create file systemwatcher with pattern */python (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Create file systemwatcher with pattern */*/python (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Register Intepreter Watcher, Class name = f, Arg 1: <Uri:/home/ryan/src/vulkan>, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Build the workspace interpreter watcher, Class name = h, Arg 1: <Uri:/home/ryan/src/vulkan>, Return Value: <Return value cannot be serialized for logging> (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = settings, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from settings (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = workspaceEnvs, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from workspaceEnvs (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detection of Python Interpreter for Command python3.7 and args  failed (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detection of Python Interpreter for Command python3.6 and args  failed (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule system is {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule currentPath is {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule windowsRegistry is nothing (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from cachedInterpreters, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, <No Resource> (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by CondaEnvFileService are of count 0 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by PipEnvService are of count 0 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by CondaEnvService are of count 0 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by VirtualEnvService are of count 0 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"}, Arg 2: <argument cannot be serialized for logging>, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = cachedInterpreters, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from cachedInterpreters (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
4console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from currentPath, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","cachedEntry":true} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by KnownPathsService are of count 10 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by CurrentPathService are of count 4 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by WorkspaceVirtualEnvService are of count 3 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","cachedEntry":true}, Arg 2: <argument cannot be serialized for logging>, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = currentPath, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from currentPath (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: true (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Hide locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: All locators have completed locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = windowsRegistry, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from windowsRegistry (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from system, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"}, Arg 2: <argument cannot be serialized for logging>, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = system, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from system (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
6console.ts:134 [Extension Host] Python Extension: autoSelectInterpreter, Class name = g, Arg 1: <Uri:/home/ryan/src/vulkan>, Arg 2: <argument cannot be serialized for logging>, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, /home/ryan/src/vulkan (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = windowsRegistry, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from windowsRegistry (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = settings, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from settings (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = workspaceEnvs, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from workspaceEnvs (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, /home/ryan/src/vulkan (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
3console.ts:134 [Extension Host] Python Extension: autoSelectInterpreter, Class name = g, Arg 1: <Uri:/home/ryan/src/vulkan>, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking support of .NET, Class name = u, , Return Value: true (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, /home/ryan/src/vulkan (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from currentPath, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from system, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule system is {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule currentPath is {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule windowsRegistry is nothing (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from cachedInterpreters, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = settings, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from settings (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = workspaceEnvs, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from workspaceEnvs (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Activation Commands received undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: getActivatedEnvironmentVariables, Class name = S, Arg 1: <Uri:/home/ryan/src/vulkan>, Arg 2: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, /home/ryan/src/vulkan (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule system is {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule currentPath is {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule windowsRegistry is nothing (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from cachedInterpreters, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"}, Arg 2: undefined, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"}, Arg 2: undefined, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"}, Arg 2: undefined, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = currentPath, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from currentPath (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = system, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from system (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = cachedInterpreters, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from cachedInterpreters (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
3console.ts:134 [Extension Host] Python Extension: autoSelectInterpreter, Class name = g, Arg 1: <Uri:/home/ryan/src/vulkan>, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"}, Arg 2: <argument cannot be serialized for logging>, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = cachedInterpreters, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from cachedInterpreters (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from currentPath, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"}, Arg 2: <argument cannot be serialized for logging>, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = currentPath, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from currentPath (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = windowsRegistry, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from windowsRegistry (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Create file systemwatcher with pattern */python (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Create file systemwatcher with pattern */*/python (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Display locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Notify locators are locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Detected refreshing of Interpreters, Class name = p, Arg 1: {}, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Register Intepreter Watcher, Class name = f, Arg 1: undefined, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Build the workspace interpreter watcher, Class name = h, Arg 1: undefined, Return Value: <Return value cannot be serialized for logging> (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from system, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by PipEnvService are of count 0 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"}, Arg 2: <argument cannot be serialized for logging>, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = system, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from system (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
6console.ts:134 [Extension Host] Python Extension: autoSelectInterpreter, Class name = g, Arg 1: undefined, Arg 2: <argument cannot be serialized for logging>, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = windowsRegistry, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from windowsRegistry (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = settings, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from settings (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = workspaceEnvs, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from workspaceEnvs (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
2console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, <No Resource> (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
3console.ts:134 [Extension Host] Python Extension: autoSelectInterpreter, Class name = g, Arg 1: undefined, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, <No Resource> (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from currentPath, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule system is {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule currentPath is {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule windowsRegistry is nothing (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from cachedInterpreters, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Selected Interpreter from system, {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"} (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Current value for rule workspaceEnvs is nothing (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Interpreters returned by WorkspaceVirtualEnvService are of count 3 (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
2console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown"}, Arg 2: undefined, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: setGlobalInterpreter, Class name = g, Arg 1: {"architecture":3,"path":"/usr/bin/python3","version":{"raw":"3.5.3-final","major":3,"minor":5,"patch":3,"prerelease":["final"],"build":[],"version":"3.5.3-final"},"sysPrefix":"/usr","fileHash":"feaae20a88b56322cdfaa812282b246ff2cf8ee9f9bc3138ea49ed05e5cd8cd2084e9dcd5298f92f01b9b674eda7c9bb0a2356cf7f5e40ee619e7c93b2375e1b","type":"Unknown","displayName":"Python 3.5.3 64-bit"}, Arg 2: undefined, Return Value: false (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = currentPath, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from currentPath (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = cachedInterpreters, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from cachedInterpreters (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Rule = system, result = runNextRule (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Executing next rule from system (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Checking whether locactors have completed locating, Class name = p, , Return Value: true (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: Hide locator refreshing progress, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
console.ts:134 [Extension Host] Python Extension: All locators have completed locating, Class name = p, , Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
3console.ts:134 [Extension Host] Python Extension: autoSelectInterpreter, Class name = g, Arg 1: undefined, Return Value: undefined (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
3console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, /home/ryan/src/vulkan (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
```

If not the first time trying debug in a session, this is all I get.

```
3console.ts:134 [Extension Host] Python Extension: Cached data exists getEnvironmentVariables, /home/ryan/src/vulkan (at r.logInformation (/home/ryan/.vscode-insiders/extensions/ms-python.python-2019.1.0/out/client/extension.js:1:16616))
```

**Comment by mjbvz** [comment]
Duplicate of #68997
